core/arch/arm/mm/tee_pager.c

```diff
@@ -770,4 +809,38 @@ void tee_pager_add_pages(vaddr_t vaddr, size_t npages, bool unmap)
 		core_tlb_maintenance(TLBINV_UNIFIEDTLB, 0);
 	}
 }
+
+void tee_pager_release_zi(vaddr_t vaddr, size_t size)
+{
+	bool unmaped = false;
+	uint32_t exceptions = thread_mask_exceptions(THREAD_EXCP_ALL);
```

Comment:
When we go multi-core these two functions will need to take a pager spinlock, for now I think it's wise to disable exceptions while changing linked lists that the pager traverses.